staging/src/k8s.io/apiserver/pkg/storage/value/encrypt/envelope/kmsv2/cache.go

```diff
@@ -39,6 +40,8 @@ type simpleCache struct {
 	// hashPool is a per cache pool of hash.Hash (to avoid allocations from building the Hash)
 	// SHA-256 is used to prevent collisions
 	hashPool *sync.Pool
+	// mu protects the set calls
+	mu sync.RWMutex
```

Comment:
I believe I was wrong on the need for a lock here so I think this can be dropped.